src/components/LatestZephyrManifest.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+import CodeBlock from "@theme/CodeBlock";
+
+export function ZephyrManifest() {
+    const [latestTag, setLatestTag] = useState("");
+
+    useEffect(() => {
+        const fetchLatestTag = async () => {
+            try {
+                const response = await fetch(
+                    "https://api.github.com/repos/memfault/memfault-firmware-sdk/releases/latest"
+                );
+                const data = await response.json();
+                const tag = data.tag_name;
+                setLatestTag(tag);
+            } catch (error) {
+                console.error("Failed to fetch the latest tag:", error);
+            }
+        };
+
+        fetchLatestTag();
+    }, []);
+
+    const codeSnippet = `
+    manifest:
+      remotes:
+        # Add the Memfault GitHub repo
+        - name: memfault
+          url-base: https://github.com/memfault
+
+      projects:
+        # The Memfault SDK
+        - name: memfault-firmware-sdk
+          path: modules/lib/memfault-firmware-sdk
+          revision: ${latestTag}
+          remote: memfault
+  `;
+
+    // from https://docusaurus.io/docs/next/markdown-features/code-blocks#usage-in-jsx
+    return (
+        <div>
+            <CodeBlock language="yaml" title="west.yml">
+                {codeSnippet}
+            </CodeBlock>
+        </div>
```

Comment:
Mild case of `div`itis :grin: 
```suggestion
          <CodeBlock language="yaml" title="west.yml">
              {codeSnippet}
          </CodeBlock>
```